common/menu-entry.c

```diff
@@ -490,6 +490,14 @@ bool menuEntryLoad(menuEntry_s* me, const char* name, bool shortcut) {
         return false;
     }
 
+    //check for .filename.star in same path
+    strptr = getSlash(me->path);
+    if (strptr[0] == '/') strptr++;
+    int strptrLen = strlen(strptr);
+    snprintf(tempbuf, sizeof(tempbuf)-1, "%.*s.%.*s.star", (int)(strlen(me->path) - strptrLen), me->path, (int)strptrLen, strptr);
```

Comment:
Change output to `me->starpath` and remove the below strcpy.

Reply:
Done.